monitoring/README.md

```diff
@@ -3,44 +3,101 @@
 <img src="images/ks-monitoring-setup.png" width="80%" height="80%" title="kubestellar-monitoring-setup">
 
 
+### Requirement
+To follow the instructions here, you need to have release `v0.2.0-rc12` or higher for the status-addon controller and agent installed in your enviroment. If you're using an older release, please follow these instructions to update your environment: 
+
+1.  Remove the status-addon-controller and status-addon-agent from your current environment:
+
+```bash
+helm --kube-context its1 -n open-cluster-management delete status-addon
+k --context its1 -n cluster1 delete manifestwork addon-addon-status-deploy-0
+```
+
+2. Re-deploy the status-addon controller and agents: 
+
+```bash
+helm --kube-context its1 upgrade --install ocm-status-addon -n open-cluster-management oci://ghcr.io/kubestellar/ocm-status-addon-chart --version v0.2.0-rc12
+```
+
 ### Description
```

Comment:
The reader should be told what versions of KubeStellar this is intended to work with. Release 0.24.0? Branch `main`?

Reply:
I added the following information in the README: 
```
To follow the instructions here you need to use KubeStellar main branch for any future KubeStellar release higher than `0.24.0`
```
This is just a point in time instructions. Ideally, these instructions will be intended for specific release of KS - to conduct performance analysis for a specific release of KS. However, right now it works also with `main` - but this branch changes constantly.